recipes-kernel/linux/linux-nilrt-debug_git.bb

```diff
@@ -12,7 +12,8 @@ SRC_URI += "\
 
 # This is the place to overwrite the source AUTOREV from linux-nilrt.inc, if
 # the kernel recipe requires a particular ref.
-#SRCREV = ""
+SRCREV = "378d8fbcc32082ee008a0df967ebd0f41a182877"
+SRCREV_xilinx-zynq = "051c9569fc919a173fbc7a56c75efdbba3b13b8c"
```

Comment:
Is this necessary for the hardknott branches?

Reply:
Whatever we did last release is probably fine.

Reply:
Yep, that's what I did here. Resolving this.